### PR TITLE
Category filehandle.c fix

### PR DESCRIPTION
**Pull request by AlbertHald**
Programmet indlæser nu category fra recipe ind i stedet for weight.

### CODE REVIEW

File: func/definitions.h [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #define FILE_CATEGORY_DELIMITER "."
 #define MAX_FILE_LINE_LENGTH 128
 #define MAX_RECIPE_NAME 128
+#define MAX_INGREDIENT_CATEGORY 128
```
Comment (atjn) [inline func/definitions.h:7]:
```suggestion
#define MAX_INGREDIENT_CATEGORY_NAME 128
```
Consider calling this something with name, to make it easier to understand what it does :)

File: func/definitions.h [hunk 2]
```diff
@@ -15,7 +16,7 @@
 typedef struct {
    char name[MAX_INGREDIENT_NAME];
    float amount;
-   unsigned short weight;
+   char ingredientCategory[MAX_INGREDIENT_CATEGORY];
```
Comment (atjn) [inline func/definitions.h:19]:
```suggestion
   char ingredientCategory[MAX_INGREDIENT_CATEGORY_NAME];
```
☝️

**Review by BaBrixx — APPROVED** [review]
LVGTM TYVM
Unknown

**Comment by TSKsmiley** [comment]
image

**Review by TSKsmiley — APPROVED** [review]

**Pull request merged**